### PR TITLE
WebGPURenderer: Support multiple render targets

### PR DESCRIPTION
**Pull request by aardgoose**
Related issue: #26801

A little bit hacky with the `isOutputStructVar` flag used to omit code insertion and member name prefixing.

### CODE REVIEW

File: examples/jsm/nodes/core/OutputStructNode.js [hunk 1]
```diff
@@ -33,15 +33,19 @@ class OutputStructNode extends Node {
 	generate( builder, output ) {
 
 		const nodeVar = builder.getVarFromNode( this, this.nodeType );
-		const propertyName = builder.getPropertyName( nodeVar );
+		nodeVar.isOutputStructVar = true;
+
+		let propertyName = builder.getPropertyName( nodeVar );
```
Comment (LeviPesin) [inline examples/jsm/nodes/core/OutputStructNode.js:38]:
```suggestion
		const propertyName = builder.getPropertyName( nodeVar );
```

File: examples/jsm/nodes/core/OutputStructNode.js [hunk 2]
```diff
@@ -33,15 +33,19 @@ class OutputStructNode extends Node {
...
 
 		const members = this.members;
 
-		for ( let i = 0; i < members.length; i++ ) {
+		const structPrefix = propertyName !== '' ? propertyName + '.' : propertyName;
```
Comment (LeviPesin) [inline examples/jsm/nodes/core/OutputStructNode.js:42]:
```suggestion
		const structPrefix = propertyName !== '' ? propertyName + '.' : '';
```

**Comment by sunag** [comment]
I think that the PR https://github.com/mrdoob/three.js/pull/26801 introduced some conflicts.

**Comment by aardgoose** [comment]
Fixed now

**Pull request merged**